toucan_connectors/mongo/mongo_connector.py

```diff
@@ -220,6 +221,25 @@ def get_slice(
             total_count = len(df)
         return DataSlice(df, total_count)
 
+    def get_df_with_regex(
+        self,
+        data_source: MongoDataSource,
+        field: str,
+        regex,
```

Comment:
```suggestion
        regex: str,
```
Obvious but it costs nothing ;) Is there a default to `*` or something?

Reply:
(Don't hesitate to push force and rewrite the whole history btw)